frontend/src/Pages/AuthenticationPage/ChangePasswordPage.jsx

```diff
@@ -85,6 +86,11 @@ export function ChangePasswordPage() {
                 required: true,
                 message: PASSWORD_REQUIRED,
               },
+              {
+                min: 8,
+                max: 16,
+                message: PASSWORD_RANGE_FROM_8_TO_16_CHARACTERS,
```

Comment:
Redundant comma

Reply:
Redundant comma is auto generated by Format code Prettier